ftfy/fixes.py

```diff
@@ -325,12 +325,12 @@ def fixup(match):
             except ValueError:
                 pass
         else:
-            # named entity
+            # This is a named entity; if it's a known HTML5 entity, replace
+            # it with the appropriate character.
             try:
-                text = entities[text[1:]]
+                return entities.html5[text[1:]]
             except KeyError:
-                pass
-        return text  # leave as is
```

Comment:
I'm confused as to the removal of this line -- isn't it needed by the fall-through from 326 above?